manifests/s/SaltStack/SaltMinion/3003.1/SaltStack.SaltMinion.locale.en-US.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: SaltStack.SaltMinion
+PackageVersion: 3003.1
```

Comment:
```suggestion
PackageVersion: 30.03.1
```

Reply:
This will break upgrades since the other copies of the minion are using the usual (marketing) version number scheme. Do we want to change those too?